Unity-Technologies/usd-unity-sdk
# [USDU-245] Adds Test case for EditorOnly tag object not being exported

## Purpose of this PR

**Ticket/Jira #:**

[USDU-245](https://jira.unity3d.com/browse/USDU-245)
Objects tagged as "EditorOnly" are still exported #303

## Testing

ExportObjectWithEditorOnlyTag_DoesNotExportEditorOnly
1. Create a root object
2. Creates a default object - set it as 1st child of root object
3. Creates an EditorOnly Tag object - set it as 2nd child of root object
4. Exports it as a USD file
5. Open and iterate over the exported USD file
6. Make sure the EditorOnly Tag child object does not exist

**Complexity:**
1

**Halo Effect:**
1

## Additional information

**Note to reviewers:**

Currently set as Ignore due to [USDU-245](https://jira.unity3d.com/browse/USDU-245)
Will remove the ignore attribute once ticket is solved

**Reminder:**
